ngraph/core/src/op/gelu.cpp

```diff
@@ -63,7 +35,7 @@ shared_ptr<Node> op::v0::Gelu::clone_with_new_inputs(const OutputVector& new_arg
     return make_shared<op::v0::Gelu>(new_args.at(0));
 }
 
-void op::v0::Gelu::pre_validate_and_infer_types() {
+void op::v0::Gelu::validate_and_infer_types() {
```

Comment:
shouldn't be NGRAPH_OP_SCOPE added here?

Reply:
Ok, Done (for HardSigmoid also)